src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -1,26 +1,28 @@
 /**
  * Copyright 2023 IBM Corp. All Rights Reserved.
- *
+ * <p>
```

Comment:
```suggestion
 *
```

Unrelated to this specific PR